### ISSUE TITLE
6.6.2 Error in redirect class after 3DS authorisation

### ISSUE THREAD

**Issue by freestream** [Adyen/adyen-magento2#861]
**Describe the bug**
After a customer completes the 3DS authorisation (in this specific case with `Mobilt BankID`) the customer ends up at a redirect controller in your module after purchasing with credit card. 
We launched Magento 2 yesterday at that point we had version `6.5` of `adyen-magento2`. We experienced some fatal crashes of the `adyen/transparent/redirect` controller. I was unable to obtain any error from this crash.

I noticed that there was a never version of `adyen-magento2` so I updated it to `6.6.2`. The same issue still remains but now i actually get logging of the error.

```
[error] 39#39: *240 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught Error: Class 'Adyen\Service\Validator\DataArrayValidator' not found in /var/www/magento/vendor/adyen/module-payment/Block/Transparent/Redirect.php:73
Stack trace:
#0 /var/www/magento/vendor/adyen/module-payment/view/frontend/templates/transparent/redirect.phtml(25): Adyen\Payment\Block\Transparent\Redirect->getPostParams()
#1 /var/www/magento/vendor/magento/framework/View/TemplateEngine/Php.php(59): include('/var/www/magent...')
#2 /var/www/magento/vendor/magento/framework/View/Element/Template.php(271): Magento\Framework\View\TemplateEngine\Php->render(Object(Adyen\Payment\Block\Transparent\Redirect), '/var/www/magent...', Array)
#3 /var/www/magento/vendor/magento/framework/View/Element/Template.php(301): Magento\Framework\View\Element\Template->fetchView('/var/www/magent...')
#4 /var/www/magento/vendor/magento/framework/View/Element/AbstractBlock.php(1100): Magento\Framework\View\Element\Template->_toHtml()
#5 /var/www/magento/vendor/magento/framework/View"
```

**To Reproduce**

https://github.com/Adyen/adyen-magento2/blob/6.6.2/Block/Transparent/Redirect.php#L26

According to the module its satisfied by version `6.3` of `adyen-php-api-library` but that versionen does not have the class that is used in the class above.
https://github.com/Adyen/adyen-magento2/blob/6.6.2/composer.json#L17
https://github.com/Adyen/adyen-php-api-library/tree/6.3.0/src/Adyen/Service

It where first introduced in version `7.1.0`
https://github.com/Adyen/adyen-php-api-library/tree/7.1.0/src/Adyen/Service

**Comment by acampos1916** [comment]
Hello @freestream, thanks for reporting this issue,

Indeed we have confirmed this problem in the `6.6.2` version and we're working on an urgent fix. We'll update this issue when a solution has been released.

Cheers.
Ángel

**Comment by AlexandrosMor** [comment]
Hello @freestream,

We include the fix in the latest release 6.6.3 which is available on [github releases](https://github.com/Adyen/adyen-magento2/issues/861). It would be great if you can try it and let us know about the outcome.

kind regards,
Alexandros
